frontend/apps/reader/modules/readerlink.lua

```diff
@@ -716,7 +716,7 @@ end
 function ReaderLink:onAddCurrentLocationToStack(show_notification)
     self:addCurrentLocationToStack()
     if show_notification then
-        Notification:notify(_("Current location added to history."))
+        Notification:notify(_("Current location added to history."), Notification.SOURCE_ALWAYS_SHOW)
```

Comment:
Am I right that you want to override my settings and enforce showing me the notification?

Reply:
I don't "want" to do anything, it just won't show it otherwise when triggered with shortcuts.

Reply:
It should depend on the user settings. Why it doesn't?

Reply:
take two, coming up